docs/UserGuide.md

```diff
@@ -88,6 +88,8 @@ Before you go on, here are some key terms and symbols you should know!
 #### <div> :bulb: </div>
 - The light bulb symbol will show you a tip to make your Projact experience smoother.
 
+#### <div> :exclamation: </div>
+- The exclamation mark symbol is a warning for any side effects that may result from the command, or any other important information you should know before you proceed.
```

Comment:
should we standardise all user input constraints to either information source or exclamation?

Reply:
I was thinking the information source to be used for other purposes? Like additional information. But exclamation is like something that it is important to take note of at that point in time.